packages/react-ui-kit/src/Layout/StyledApp.tsx

```diff
@@ -23,11 +23,12 @@ import {CSSObject, jsx} from '@emotion/react';
 
 import {GlobalStyle} from '../GlobalStyle';
 import {filterProps} from '../util';
-import {THEME_ID, Theme, ThemeProvider} from './Theme';
+import {THEME_ID, Theme, ThemeProvider, themes} from './Theme';
 
 export interface StyledAppContainerProps<T = HTMLDivElement> extends React.HTMLProps<T> {
   backgroundColor?: string;
   themeId?: THEME_ID;
+  theme?: Theme;
 }
```

Comment:
You haven't solved my challenge yet ;) 
I'd like this type to only allow for a `themeId` or a `theme` but not both ;)